src/main/java/com/opentable/components/filterorder/OrderDeclaration.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.opentable.components.filterorder;
+
+import javax.servlet.Filter;
+
+/**
+ * Inject {@link OrderDeclaration} instances to assert the order in which filters should be run in the servlet
+ * container.  Either specify that a filter comes last (there may be only one of these), or that a filter
+ * <em>depends on</em> another (i.e., a filter must come after another).
+ *
+ * <p>
+ * Note that this implementation uses the {@link Filter} classes themselves to declare dependencies. So if there are
+ * multiple filters in your chain of the same class (e.g., with different configurations), this limits the
+ * expressiveness of this library.
+ *
+ * @see FilterOrderResolver
+ */
+public final class OrderDeclaration {
+    final boolean last;
```

Comment:
Should we use private fields and accessor methods?

Reply:
I don't think so.  It's an internal API, so we don't gain any future-proofing.  Just more Java-ey boilerplate!